parser/src/lexer.rs

```diff
@@ -457,7 +457,13 @@ where
                 break;
             }
         }
-        u8::from_str_radix(&octet_content, 8).unwrap() as char
+        match u32::from_str_radix(&octet_content, 8) {
```

Comment:
I think you should be able to just `unwrap()` this, since we just verified that it's all octal digits